.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+<!-- Steps to reproduce the behavior -->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Environment**
+<!-- Information about the environment. -->
+
+* **Operation system:**
+* **Node version:**
```

Comment:
Let's give the users some commands to run figure these out, like `node -v`.

Reply:
Since the issue will usually happen in production, it's not trivial to run even such a small command on the machine/container. I would expect the Node version to be known.

Reply:
How about the tracer and agent version?

Reply:
In general I expect users to know what they are using in their stack, or how to find out otherwise. Explaining exactly how to get this information could get complicated if we want the explanation to be really helpful.